pkg/wpa/passphrase/passphrase.go

```diff
@@ -0,0 +1,36 @@
+package passphrase
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"fmt"
+
+	"golang.org/x/crypto/pbkdf2"
+)
+
+const ResultFormat = `network={
+	ssid="%s"
+	#psk="%s"
+	psk=%s
```

Comment:
The c impl is using printf %02 and iterating through the result. Is there a possible bug here by doing plain string insertion?

Reply:
I don't think it will introduce any bug into this program because in hex.EncodeToString, they do something similar to printf %02 under the hood. Also, the output of pbkdf2.Key is guaranteed to be 32 bytes, and hex.EncodeToString generates a string of 64 characters. Thus, I don't think string length will be a problem either.